src/server/shared/SharedDefines.h

```diff
@@ -105,6 +105,33 @@ enum Races
 
 #define RACEMASK_HORDE RACEMASK_ALL_PLAYABLE & ~RACEMASK_ALLIANCE
 
+// DisplayRace values from CreatureDisplayInfoExtra.dbc
+enum class DisplayRace : uint8
```

Comment:
Constants should be in uppercase.

Reply:
I used #17607 by @Kitzunu as a reference which also has no upper case constants in enum class

Reply:
> I used #17607 by @Kitzunu as a reference which also has no upper case constants in enum class

Nothing is sacred anymore 😭

Reply:
In enum we use uppercase, but not in enum class